css/style.css

```diff
@@ -0,0 +1,30 @@
+*{
+    padding: 0;
+margin: 0;
+box-sizing: border-box;
+}
```

Comment:
you put a great effort. however I would suggest to change following according to css best practice.
- reduce '2 space' before the 'padding: 0'
- add '2 space before 'margin: 0'
- add '2 space' before 'margin: 0'